app/views/content/train-to-be-a-teacher/teaching-degree-apprenticeships.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Teacher degree apprenticeships (TDAs)"
+heading: "Teacher degree apprenticeships (TDAs)"
+description: |-
+ A new route to become a teacher without a degree, a teacher degree apprenticeship (TDA) lets you work at a school while you qualify as a teacher.
+related_content:
+    Is teaching right for me? : "/is-teaching-right-for-me"
+    Train to teach if you do not have a degree : "/train-to-be-a-teacher/if-you-dont-have-a-degree"
+    Who do you want to teach? : "/is-teaching-right-for-me/who-do-you-want-to-teach"
+external_content:
+    - content/train-to-be-a-teacher/promos/find-your-undergraduate-course
+navigation: 20.20
+navigation_title: Teacher degree apprenticeships (TDAs)
+navigation_description: A TDA lets you work at a school and earn a salary while getting a degree and qualified teacher status (QTS).
+talk_to_us: false
+---
+If you do not already have an undergraduate degree, you may be eligible to do a teacher degree apprenticeship (TDA), a new route to become a qualified teacher.
+
+##  How do TDAs work?
+This new teacher training programme means you’ll work in a school and earn a salary while getting a bachelor’s degree and [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
+ 
+The TDA is a 4-year, full-time course that lets you train as either a primary or secondary teacher with an accredited teacher training provider. Trainee teachers will spend an average of 40% of their time studying for their degree with a university.
+
+The salary you’ll earn as a trainee will reflect the level of responsibility that’s suitable for each stage of your course. You do not have to pay tuition fees but you will not be eligible for student finance.
+
+The [teacher training providers confirmed so far](https://www.gov.uk/government/publications/secondary-maths-teacher-degree-apprenticeship-funding-pilot-providers?) will offer TDAs in maths, with other subjects likely to follow.
+
+## Who’s eligible to apply for a TDA?
+The TDA will be available to both people who are leaving school, or at the start of their careers, and those already working who are interested in becoming a teacher, including teaching assistants.
```

Comment:
```suggestion
The TDA will be available to both people who are leaving school and those already working who are interested in becoming a teacher, including teaching assistants.
```
Is `start of their careers` needed?

Reply:
I thought I'd already deleted that. Will do it now.